src/compat/db_admin.c

```diff
@@ -1805,6 +1805,39 @@ db_set_user_comment (DB_OBJECT * user, const char *comment)
   return (retval);
 }
 
+/*
+ * db_get_object_type() - This returns database object type of given MOP
+ * return    : DB_OBJECT_TYPE
+ * obj_(in) : an object
+ *
+ */
+// TODO: find better solution
+DB_OBJECT_TYPE
+db_get_object_type (MOP obj_)
+{
+  DB_OBJECT_TYPE ret_val = DB_OBJECT_UNKNOWN;
+
+  assert (obj_->class_mop != NULL);
+
+  OID *mop = WS_OID (obj_->class_mop);
+  if (mop == WS_OID (sm_Root_class_mop))
```

Comment:
포인터값만 체크하면 되나요?
혹시 포인터는 달라도 그 내용이 동일한 지를 체크 하려면 OID_COMPARE() 매크로 사용해야 하는데, 여기서는 필요가 없는 부분일까요?

Reply:
다른 코드에서도 OID_EQ 매크로로 비교하고 있네요. 다음과 같이 수정하겠습니다.
```suggestion
  if (OID_EQ (mop, WS_OID (sm_Root_class_mop))
```